docs/src/running-validator/validator-reqs.md

```diff
@@ -2,35 +2,32 @@
+- Disk
+  - PCIe Gen3 x4 NVME SSD, or better
+  - Accounts: 500GB, or larger. High TBW (Total Bytes Written)
+  - Ledger: 1TB or larger. High TBW suggested
+  - OS: (Optional) 500GB, or larger. SATA OK
+  - The OS may be installed on the ledger disk, though testing has shown better
+performance with the ledger on its own disk
+  - Accounts and ledger _can_ be stored on the same disk, however due to high
+IOPS, this is not recommended
+  - The Samsung 970 and 980 Pro series SSDs are popular with the validator community
 - GPUs
-  - While a CPU-only node may be able to keep up with the initial idling network, once transaction throughput increases, GPUs will be necessary
-  - What kind of GPU?
-    - We recommend Nvidia Turing and volta family GPUs 1660ti to 2080ti series consumer GPU or Tesla series server GPUs.
-    - We do not currently support OpenCL and therefore do not support AMD GPUs. We have a bounty out for someone to port us to OpenCL. Interested? [Check out our GitHub.](https://github.com/solana-labs/solana)
-- Power Consumption
-  - Approximate power consumption for a validator node running an AMD Threadripper 3950x and 2x 2080Ti GPUs is 800-1000W.
-
-### Preconfigured Setups
-
-Here are our recommendations for low, medium, and high end machine specifications:
-
-|                     | Low end                                               | Medium end             | High end               | Notes                                                                                  |
-| :------------------ | :---------------------------------------------------- | :--------------------- | :--------------------- | :------------------------------------------------------------------------------------- |
-| CPU                 | AMD Ryzen 3950x                                       | AMD Threadripper 3960x | AMD Threadripper 3990x | Consider a 10Gb-capable motherboard with as many PCIe lanes and m.2 slots as possible. |
-| RAM                 | 32GB                                                  | 64GB                   | 128GB                  |                                                                                        |
-| Ledger Drive        | Samsung 860 Evo 2TB                                   | Samsung 860 Evo 4TB    | Samsung 860 Evo 4TB    | Or equivalent SSD                                                                      |
-| Accounts Drive\(s\) | None                                                  | Samsung 970 Pro 1TB    | 2x Samsung 970 Pro 1TB |                                                                                        |
-| GPU                 | Nvidia 1660ti                                         | Nvidia 2080 Ti         | 2x Nvidia 2080 Ti      | Any number of cuda-capable GPUs are supported on Linux platforms.                      |
+  - Not strictly necessary at this time
+  - Motherboard and power supply speced to add one or more high-end gaming GPUs
+in the future suggested
```

Comment:
@t-nelson So lame me... I was completely forgetting this was the perfect time to add mention of required network bandwidth for both uplink and downlink...  Remember the support ticket.. ;)